pkg/crc/machine/machine.go

```diff
@@ -305,7 +322,8 @@ func Start(startConfig StartConfig) (StartResult, error) {
 	// If no error, return usage message
 	if result.Error == "" {
 		time.Sleep(time.Minute * 3)
-		logging.Infof("To access the cluster using 'oc', run 'eval $(crc oc-env) && oc login -u kubeadmin -p %s %s'", result.ClusterConfig.KubeAdminPass, result.ClusterConfig.ClusterAPI)
+		logging.Infof("To access the cluster, first set up your environment by following 'crc oc-env' instructions")
+		logging.Infof("Then you can access it with oc by running 'oc login -u kubeadmin -p %s %s'", result.ClusterConfig.KubeAdminPass, result.ClusterConfig.ClusterAPI)
```

Comment:
I think we have to put both the messages since `developer` user is not have cluster admin right by default so if a user really want to try out cluster with admin then `kubeadmin` is best fit a user.